amy/templates/get_involved_form.html

```diff
@@ -0,0 +1,24 @@
+{% extends "base.html" %}
+{% block navbar %}
+  {% include 'navigation_instructor_dashboard.html' %}
+{% endblock %}
+
+
+{% load crispy_forms_tags %}
+
+{% block content %}
+<p>
+  Provide details of your Get Involved activity. Your submission will be reviewed within 7-10 days. 
+  You can track your progress by returning to the <a href="{% url 'training-progress' %}">training progress</a> page.
+</p>
+<p>
+  Only submit an activity that you have already completed.
+</p>
+<p>
+  If submitting a GitHub contribution, please be sure your contribution is to a repository in one of the 
+  <a href="https://docs.carpentries.org/topic_folders/communications/tools/github_organisations.html">
+    GitHub organisations owned by The Carpentries
+  </a>.
+</p>
+{% crispy form %}
```

Comment:
Can this be replaced with the following?

```
{% include "includes/get_involved_form_embedded.html" with get_involved_form=form %}
```

Reply:
It can, though the `...embedded.html` file is a holdover from earlier experiments that I had meant to delete! Do you still think it would be better to use the `include` even though this template is fairly simple?

Reply:
@elichad If the "embedded" file is going away then there's no need to use `include` here.